scripts/qa_crowbarsetup.sh

```diff
@@ -1516,6 +1523,12 @@ function onadmin_proposal()
     if iscloudver 5plus; then
         update_one_proposal dns default
     fi
+    if [ "$networkingplugin" = vmware ] && iscloudver 5plus ; then
+        cmachines=`crowbar machines list`
+        for machine in $cmachines; do
+            ssh $machine 'zypper mr -p 90 SLE-Cloud-PTF'
```

Comment:
For SUSE Cloud 5 we have openvswitch packages with NSX patch in version 2.0. On SUSE Cloud ISO exist openvswitch packages in version 2.1.